test/e2e/case13_templatization_test.go

```diff
@@ -324,4 +367,53 @@ var _ = Describe("Test templatization", func() {
 			utils.Kubectl("delete", "configmap", configMapReplName, "-n", "default")
 		})
 	})
+	Describe("Test the copy configMap function", func() {
+		const configMapName = "configmap-copy-configmap-object"
+		const configMapReplName = configMapName + "-repl"
+		It("Should have the expected ConfigMap created", func() {
+			By("Creating the ConfigMap to reference")
+			configMap := corev1.ConfigMap{
+				ObjectMeta: v1.ObjectMeta{
+					Name: configMapName,
+				},
+				Data: map[string]string{"message": "Hello Raleigh!"},
+			}
+			_, err := clientManaged.CoreV1().ConfigMaps("default").Create(
+				context.TODO(), &configMap, v1.CreateOptions{},
+			)
+			Expect(err).To(BeNil())
+			By("Creating the configuration policy that references the ConfigMap")
+			utils.Kubectl("apply", "-f", case13CopyRefObjectYaml, "-n", testNamespace)
+
+			By("By verifying that the policy is compliant")
+			Eventually(
+				func() interface{} {
+					managedPlc := utils.GetWithTimeout(
+						clientManagedDynamic,
+						gvrConfigPolicy,
+						case13CopyRefObject,
+						testNamespace,
+						true,
+						defaultTimeoutSeconds,
+					)
+
+					return utils.GetComplianceState(managedPlc)
+				},
+				defaultTimeoutSeconds,
+				1,
+			).Should(Equal("Compliant"))
+
+			By("By verifying that the replicated ConfigMap has the expected data")
+			replConfigMap, err := clientManaged.CoreV1().ConfigMaps("default").Get(
+				context.TODO(), configMapReplName, v1.GetOptions{},
+			)
+			Expect(err).To(BeNil())
+			Expect(replConfigMap.Data["message"]).To(Equal("Hello Raleigh!"))
+		})
+		It("Cleans up", func() {
+			deleteConfigPolicies([]string{case13CopyRefObject})
```

Comment:
Is this clean up working well without "ordered", ginkgo runs parallel sometimes it disturbs other testings